aperture.go

```diff
@@ -284,37 +289,32 @@ func (a *Aperture) Start(errChan chan error) error {
 
 	log.Infof("Using %v as database backend", a.cfg.DatabaseBackend)
 
-	// Create our challenger that uses our backing lnd node to create
-	// invoices and check their settlement status.
-	genInvoiceReq := func(price int64) (*lnrpc.Invoice, error) {
-		return &lnrpc.Invoice{
-			Memo:  "LSAT",
-			Value: price,
-		}, nil
-	}
-
 	if !a.cfg.Authenticator.Disable {
+		genInvoiceReq := func(price int64) (*lnrpc.Invoice, error) {
+			return &lnrpc.Invoice{
+				Memo:  "LSAT",
```

Comment:
Non-blocking, but in the future we can insert relevant endpoint/context information here so someone can build a UI/dashboard on top of lnd/aperture to do things like track the amount earned from a given endpoint, etc. 

To do this, we'd need to expose the memo as part of the challenger interface, as only much later in the pipeline do we have the full request context information.